ca/server.go

```diff
@@ -92,6 +111,7 @@ func (s *Server) NodeCertificateStatus(ctx context.Context, request *api.NodeCer
 		s.store,
 		func(tx store.ReadTx) error {
 			node = store.GetNode(tx, request.NodeID)
+			cluster = store.GetCluster(tx, s.securityConfig.ClientTLSCreds.Organization())
```

Comment:
Oops, accidentally left this in from when I was going to add it to the cert status response. Thanks!